### PR TITLE
SYM-240: Don't use Homebrew to install pipx.

### PR DESCRIPTION
**Pull request by cky**

### CODE REVIEW

File: docs/sym.sh [hunk 1]
```diff
@@ -141,7 +137,6 @@ ensurePython38() {
 installWithPipx() {
   ensurePython38
```
Comment (jon918) [inline docs/sym.sh:138]: it looks to me like `ensurePython38` also could trigger a brew install, should there be some other logic there?

Comment (cky) [reply docs/sym.sh:138]: It'd trigger if there is no pyenv. I guess the only alternative (other than trying to use `apt-get` or the like) if there is no pyenv, is to install pyenv. Is that better than using Homebrew? What do you think?

Comment (cky) [reply docs/sym.sh:138]: We could just bail and tell the user to install Python, too.

File: docs/sym.sh [hunk 2]
```diff
@@ -117,12 +117,8 @@ ensureBrew() {
 
 ensurePipx() {
   if ! hasCommand pipx; then
-    if hasCommand brew; then
-      brew install pipx
-    else
-      $(getPythonPath) -m pip install --user pipx
-      $(getPythonPath) -m pipx ensurepath
-    fi
+    $(getPythonPath) -m pip install --user pipx
```
Comment (yasyf) [inline docs/sym.sh:120]: if they have brew we should use it, as it is the recommended way to install

File: docs/sym.sh (hunk 1 above)
Comment (yasyf) [reply docs/sym.sh:138]: I think its fine for now but maybe we confirm with the user before continuing

File: docs/sym.sh (hunk 2 above)
Comment (cky) [reply docs/sym.sh:120]: If I read you correctly, then the original code was already doing the right thing, right? Or did I miss something?

File: docs/sym.sh (hunk 1 above)
Comment (cky) [reply docs/sym.sh:138]: Just to clarify, does this sound right?

1. Check for Python 3.8. If it exists, return success.
2. Check for pyenv. If it exists, install Python 3.8 using pyenv, and return success if successful.
3. Check for Homebrew. If it exists, prompt the user to confirm installing Python 3.8 via Homebrew, and return success if the user says yes and the installation succeeded.
4. Return failure.

File: docs/sym.sh (hunk 2 above)
Comment (yasyf) [reply docs/sym.sh:120]: oops yup you're totally right. I read the future and did this 11 days ago: https://github.com/symopsio/homebrew-tap/commit/ac3a8be9c9eaa34e1596bfd03ed2ca45aec912cb#diff-81232f0b8715ed74b74a0539a216de91

File: docs/sym.sh (hunk 1 above)
Comment (yasyf) [reply docs/sym.sh:138]: yup!

File: docs/sym.sh (hunk 2 above)
Comment (yasyf) [reply docs/sym.sh:120]: totally my bad for not paying attention when I created the ticket!

**Comment by cky** [comment]
The Homebrew installation of pipx bit is already implemented, and we're punting on the pyenv stuff to next sprint. So I'll close this PR.

**Pull request closed without merging**